graphs/energy/edges/households/households_space_heater_hybrid_crude_oil_heatpump_air_water_electricity_aggregator-households_useful_demand_for_space_heating_apartments_1945_1964@useable_heat.ad

```diff
@@ -0,0 +1,6 @@
+- type = share
+- reversed = false
+
+
+
+
```

Comment:
@KoenvanB / @kaskranenburgQ why do we have all these trailing returns in the edge files?

Reply:
I think this is because the files have been copy-pasted from the other technologies. These have trailing whitespaces in them.